streams/do.m3u

```diff
@@ -508,19 +519,21 @@ https://cnn.essastream.com:3387/live/xtratvlive.m3u8
 #EXTINF:-1 tvg-id="",Xtrema Kids TV
 https://627bb251f23c7.streamlock.net:444/ExtremaKids/ExtremaKids/playlist.m3u8
 #EXTINF:-1 tvg-id="XtremoChannel.do",Xtremo Channel (720p)
-https://ss9.domint.net:3002/xtr_str/xtremochannel/playlist.m3u8
+https://ss2.tvrdomi.com:1936/xtremochannel/xtremochannel/playlist.m3u8

```

Comment:
Broken link.

Reply:
Could you please indicate what approximate hours the broadcast is running so we can see how it works?

Reply:
It's working now